public/lnd-child-process.js

```diff
@@ -65,6 +65,8 @@ module.exports.startLndProcess = async function({
     '--autopilot.private',
     '--autopilot.minconfs=0',
     '--autopilot.allocation=0.95',
+    '--autopilot.heuristic=externalscore:1',
+    '--autopilot.heuristic=preferential:0',
```

Comment:
I'd like to rely 100% on external scores for now to test how reliable/deterministic the UX is.